pkgs/by-name/ro/root/package.nix

```diff
@@ -137,9 +137,6 @@ stdenv.mkDerivation rec {
       # Eliminate impure reference to /System/Library/PrivateFrameworks
       substituteInPlace core/macosx/CMakeLists.txt \
         --replace-fail "-F/System/Library/PrivateFrameworks " ""
-      # Just like in libpng/12.nix to build the builtin libpng on macOS
```

Comment:
Thanks for removing that! The whole point of the patch release was actually to make ROOT compatible with the new Apple Clang, which also meant that this patch is not necessary anymore